frontend/src/scenes/notebooks/Notebook/notebooksListLogic.ts

```diff
@@ -156,5 +228,19 @@ export const notebooksListLogic = kea<notebooksListLogicType>([
                 })
             },
         ],
+        searchFilteredNotebooks: [
```

Comment:
This might be time to move searching to the backend...
Feels a bit rough that we filter somethings via the API and others via the frontend. Is it perhaps cleaner (and less code) if we just swap to having proper search via the api?

Reply:
Quickly looked at this (pushed commented/sipped code) so you can see what I did

The nested structure means we would be searching across all the text in the JSON. E.g. searching "paragraph" would match `{"type": "paragraph"}` or indeed `{"paragraph": "type"}`

----

And (less importantly) templates are only in the FE so we'd have to hide templates when searching or search in the FE anyway

Reply:
🙈 🤯  I can probably just search the title to copy the FE for now